README.md

```diff
@@ -1,16 +1,137 @@
 # Helium-Java
 
+# Build a Docker containerized, secure Java Web API application using Managed Identity, Key Vault, and Cosmos DB that is designed to be deployed to Azure App Service or AKS
+
+![License](https://img.shields.io/badge/license-MIT-green.svg)
+![Docker Image Build](https://github.com/retaildevcrews/helium-typescript/workflows/Docker%20Image%20Build/badge.svg)
+
+This is a Java Spring Boot Web API reference application designed to "fork and code" with the following features:
```

Comment:
should we call out web flux?

Reply:
Yes - I have documented building and running in a container.  Is there more I should put in or document?